scripts/staging/fm_regression.dml

```diff
@@ -0,0 +1,49 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+/**
+ * Factorization Machines for Regression.
+ */
+
+# Imports
+source("staging/fm.dml") as fm
+source("nn/layers/l2_loss.dml") as l2_loss
+
+train = function(matrix[double] X)
+    return (matrix[double] out) {
+   /*
+    * Trains a FM model.
+    */
```

Comment:
similar to the classification model, but use `l2_loss`, and no sigmoid layer, i.e., just pass the outputs of `fm::forward` to the loss.